apps/web/pages/order/success.vue

```diff
@@ -1,38 +1,140 @@
 <template>
-  <div class="px-4 md:px-0" data-testid="order-success-page" v-if="orderData.order">
-    <div
-      class="border border-1 border-neutral-200 mt-10 mb-20 rounded p-4 md:p-6 flex flex-col items-center max-w-2xl mx-auto"
-    >
+  <div class="px-4 md:px-0 flex items-center flex-col" data-testid="order-success-page" v-if="orderData.order">
```

Comment:
is there any case where we don't have order prop. in orderData?

Reply:
Yes, if you go to the page without order something before.